omnibus/config/software/system-probe.rb

```diff
@@ -8,6 +8,7 @@
 dependency 'libbcc'
 
 build do
-  command "#{ENV['S3_CP_CMD']} #{ENV['S3_ARTIFACTS_URI']}/system-probe.#{ENV['PACKAGE_ARCH']} #{install_dir}/embedded/bin/system-probe"
-  command "chmod 755 #{install_dir}/embedded/bin/system-probe"
```

Comment:
chmod no longer needed?

Reply:
That’s because I moved the `chmod` to the `.gitlab-ci.yml` job just after the `s3 cp` command.
The thing is that it’s S3 which is loosing the execution bit.
If the user generates `system-probe` in one container and uses it in another one to build the agent and uses a docker volume to share `system-probe` between the two, no `chmod` is needed anywhere.
It’s really the S3 copy which is loosing it. That’s why I kept the `chmod` close to the `s3 cp` command and moved the two out of omnibus and put both in `.gitlab-ci.yml`.

Reply:
Make sense now 👍